firmware/Core/Src/telecommands/telecommand_definitions.c

```diff
@@ -25,6 +25,8 @@
 #include "timekeeping/timekeeping.h"
 #include "littlefs/littlefs_helper.h"
 #include "stm32/stm32_reboot_reason.h"
+#include "telecommands/eps_telecommands.h"
```

Comment:
Please revert autoformatter changes on this file.

Reply:
Done